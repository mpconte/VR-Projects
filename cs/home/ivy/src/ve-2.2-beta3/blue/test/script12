#
# Check functions in expressions
#
set x [expr int(0)]
[stdout] writeln $x
set x [expr int(12.2)]
[stdout] writeln $x
set x [expr int(4.8)]
[stdout] writeln $x
set x [expr 5/2]
[stdout] writeln $x
set x [expr float(5)/2]
[stdout] writeln $x
