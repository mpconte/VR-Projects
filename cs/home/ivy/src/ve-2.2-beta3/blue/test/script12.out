0
12
4
2
2.5
result = ok
